main-command/src/main/scala/sbt/Watched.scala

```diff
@@ -103,8 +112,127 @@ object Watched {
   val PollDelay: FiniteDuration = 500.milliseconds
   val AntiEntropy: FiniteDuration = 40.milliseconds
   def isEnter(key: Int): Boolean = key == 10 || key == 13
-  def printIfDefined(msg: String) = if (!msg.isEmpty) System.out.println(msg)
+  def printIfDefined(msg: String): Unit = if (!msg.isEmpty) System.out.println(msg)
+
+  type Task = () => State
```

Comment:
The term task mean something already so please pick some other term here.
It would be good to add Scaladoc comment on what it is.

Reply:
Fair point. I might also be able to remove this alias anyway. I think I was mainly using it so that I could avoid the extra parens in `type Setup = (State, Watched, String) => (State, Logger, (() => State))`